index.js

```diff
@@ -64,5 +63,4 @@ window.onload = () => {
     const dt =DateTime.now(); 
   date.innerHTML = dt.toLocaleString(DateTime.DATETIME_FULL);
 
```

Comment:
- [ ] In this file kindly refactor the following functions to use the arrow syntax based on the ES6 requirement.

  ![image](https://user-images.githubusercontent.com/36542986/212639708-dd2cbdd2-469a-41e8-9f65-aec3682623de.png)

Reply:
Sure. I will do that